models/issue_comment.go

```diff
@@ -75,23 +79,24 @@ const (
 
 // Comment represents a comment in commit and issue page.
 type Comment struct {
-	ID             int64 `xorm:"pk autoincr"`
-	Type           CommentType
-	PosterID       int64 `xorm:"INDEX"`
-	Poster         *User `xorm:"-"`
-	IssueID        int64 `xorm:"INDEX"`
-	LabelID        int64
-	Label          *Label `xorm:"-"`
-	OldMilestoneID int64
-	MilestoneID    int64
-	OldMilestone   *Milestone `xorm:"-"`
-	Milestone      *Milestone `xorm:"-"`
-	OldAssigneeID  int64
-	AssigneeID     int64
-	Assignee       *User `xorm:"-"`
-	OldAssignee    *User `xorm:"-"`
-	OldTitle       string
-	NewTitle       string
+	ID               int64 `xorm:"pk autoincr"`
+	Type             CommentType
+	PosterID         int64 `xorm:"INDEX"`
+	Poster           *User `xorm:"-"`
+	IssueID          int64 `xorm:"INDEX"`
+	LabelID          int64
+	Label            *Label `xorm:"-"`
+	OldMilestoneID   int64
+	MilestoneID      int64
+	OldMilestone     *Milestone `xorm:"-"`
+	Milestone        *Milestone `xorm:"-"`
+	OldAssigneeID    int64
+	AssigneeID       int64
+	Assignee         *User `xorm:"-"`
+	OldAssignee      *User `xorm:"-"`
+	OldTitle         string
+	NewTitle         string
+	DependentIssueID int64
```

Comment:
You don't need this too.

Reply:
Yes i do?

Reply:
There is 2 places with `DependentIssueID`. One is definition of `comment` and second is when you set it to new comment variable...

Reply:
Yup. Thats why i need it.

Reply:
I saw that i was wrong with removing `LoadDepIssueDetails()`, which is why I reimplemented it in `752406c`.

I used this to get the Dependent issue details displayed in the comments. But to know which issue i need to get informations about, I also need the ID of the Dependent Issue. And that's where `DependentIssueID` comes in.

Reply:
I've probably started the featue and then forget about it, which is why there was this function (and the corresponding `DependentIssue` in the `Comment` struct). But i think this is the "cleander" solution instead of saving the titel of the dependent issue with the comment. The new way ensures also the Issues' title will always be the right one, even if it was changed after the comment was saved.